include/chemfiles/capi/frame.h

```diff
@@ -322,6 +322,14 @@ CHFL_EXPORT chfl_status chfl_frame_remove_bond(
     CHFL_FRAME* frame, uint64_t i, uint64_t j
 );
 
+/// Remove all existing bond, angles, dihedral angles and improper dihedral
```

Comment:
Sorry for being pedantic, but bond*s* (plural) sounds better :P

Reply:
thanks! that's the point of review too =)